utils/tfhe-versionable/src/lib.rs

```diff
@@ -632,6 +632,7 @@ impl Unversionize for () {
 
 impl NotVersioned for () {}
 
+// TODO: use a macro for more tuple sizes
```

Comment:
I guess you planned to implement that only if we need it at some point

Reply:
yes since we don't need it I didn't make it a priority